pkg/apiserver/utils/error.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/joomcode/errorx"
+)
+
+var (
+	ErrNS                    = errorx.NewNamespace("error.api")
+	ErrOther                 = ErrNS.NewType("other")
+	ErrUnauthorized          = ErrNS.NewType("unauthorized")
+	ErrInsufficientPrivilege = ErrNS.NewType("insufficient_privilege")
```

Comment:
Seems the `unauthorized` and `insufficient_privilege` don't make sense in Chaos Mesh. WDYT?